services/fdb/client/client.go

```diff
@@ -3461,3 +3566,193 @@ func (p *fdbServerVersionCmd) Execute(ctx context.Context, f *flag.FlagSet, args
 
 	return retCode
 }
+
+const fdbMoveDataCLIPackage = "fdbmovedata"
+
+func GetSubpackage(f *flag.FlagSet) *subcommands.Commander {
```

Comment:
do we need to add (*fdbMoveDataCmd) after `func`? 
i.e. `func (*fdbMoveDataCmd)  GetSubpackage(f *flag.FlagSet) *subcommands.Commander {`